src/k8s/pkg/k8sd/features/calico/chart.go

```diff
@@ -35,4 +35,10 @@ var (
 	calicoCtlImage = "ghcr.io/canonical/k8s-snap/calico/ctl"
 	// calicoCtlTag represents the tag to use for the calicoctl image.
 	calicoCtlTag = "v3.28.0"
+
+	// encapsulation represents the default encapsulation method to use for Calico.
+	encapsulation = "VXLAN"
```

Comment:
just so it reads better in places in the code, let's call this `defaultEncapsulation`

same below, `defaultAPIServerEnabled`